WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1155,6 +1155,16 @@ FormAddRemoveButton          = @FormButtonHeight@
 FormButtonStyle              = @cssShadow@:0px 0px 0px transparent; padding:5px; @ActiveCursor@ @PanelBorder@ @MediumRoundCorners@ background-color:@WidgetBgColor@; @ButtonText@
 
 ###################
+#   SHORTCUT LIST
+###################
+PrefShortcutLabel            = min-width: 50%; margin-bottom: 4px; @FontSize-big@ @Text@
```

Comment:
Some of the properties used here, in this block, have space after  : and ;
While others donot have.
Can we have uniform indentation for property and values